vpr/src/route/rr_graph_uxsdcxx_serializer.h

```diff
@@ -801,7 +801,8 @@ class RrGraphSerializer final : public uxsd::RrGraphBase<RrGraphContextTypes> {
     }
 
     inline unsigned int get_node_capacity(const t_rr_node& node) final {
```

Comment:
How much is this function used?  We could deprecate it and replace the call sites, since it just  gets an id from the node and then calls another function.

Reply:
I have now removed it. I added `include "globals.h"` to the top of some of the files in order to access the rr_graph, so hopefully that won't cause any issues. I am going to run qor testing again now.